src/m365/spo/commands/customaction/customaction-remove.spec.ts

```diff
@@ -162,6 +163,106 @@ describe(commands.CUSTOMACTION_REMOVE, () => {
     }), new CommandError(`Multiple user custom actions with title 'YourAppCustomizer' found. Please disambiguate using IDs: a70d8013-3b9f-4601-93a5-0e453ab9a1f3, 63aa745f-b4dd-4055-a4d7-d9032a0cfc59`));
   });
 
+  it('handles selecting single result when multiple custom actions sets with the specified name found and cli is set to prompt', async () => {
+    sinon.stub(request, 'get').callsFake((opts) => {
+      if (opts.url === "https://contoso.sharepoint.com/_api/Web/UserCustomActions?$filter=Title eq 'Places'") {
+        return Promise.resolve({
+          value: [
+            {
+              ClientSideComponentId: 'b41916e7-e69d-467f-b37f-ff8ecf8f99f2',
+              ClientSideComponentProperties: "'{testMessage:Test message}'",
+              CommandUIExtension: null,
+              Description: null,
+              Group: null,
+              HostProperties: '',
+              Id: 'a70d8013-3b9f-4601-93a5-0e453ab9a1f3',
+              ImageUrl: null,
+              Location: 'ClientSideExtension.ApplicationCustomizer',
+              Name: 'Places',
+              RegistrationId: null,
+              RegistrationType: 0,
+              Rights: [Object],
+              Scope: 3,
+              ScriptBlock: null,
+              ScriptSrc: null,
+              Sequence: 0,
+              Title: 'Places',
+              Url: null,
+              VersionOfUserCustomAction: '16.0.1.0'
+            },
+            {
+              ClientSideComponentId: 'b41916e7-e69d-467f-b37f-ff8ecf8f99f2',
+              ClientSideComponentProperties: "'{testMessage:Test message}'",
+              CommandUIExtension: null,
+              Description: null,
+              Group: null,
+              HostProperties: '',
+              Id: '63aa745f-b4dd-4055-a4d7-d9032a0cfc59',
+              ImageUrl: null,
+              Location: 'ClientSideExtension.ApplicationCustomizer',
+              Name: 'Places',
+              RegistrationId: null,
+              RegistrationType: 0,
+              Rights: [Object],
+              Scope: 3,
+              ScriptBlock: null,
+              ScriptSrc: null,
+              Sequence: 0,
+              Title: 'Places',
+              Url: null,
+              VersionOfUserCustomAction: '16.0.1.0'
+            }
+          ]
+        });
+      }
+
+      return Promise.reject(`Invalid request`);
```

Comment:
Promise.reject? Should we write the functions right Async at once?

Reply:
at the time of writing this,  the command was not still refactored to async/await. but it is now 🙂 I will update it. Good catch thanks